site/_posts/2020-08-19-securing-react-native-with-oauth.md

```diff
@@ -52,137 +53,96 @@ Next up, let's configure the OAuth server and set up our coding environment.
 
-Make sure to register your user to the new application. Doing so creates a relationship between a user and the newly created application. 
+Make sure to register your user to the new application. Doing so creates a relationship between a user and the newly created application.
 
 {% include _image.liquid src="/assets/img/blogs/react-native-oauth/fusionauth-dashboard-register-user.png" alt="Registering your user to the React Native FusionAuth application." class="img-fluid" figure=false %}
 
 If you want, you can add more users in the "Users" tab, but make sure to register them with your new application. Now, we move on to setting up the React Native project.
 
 ## Setting up the React Native development environment
 
-Since we are going to use the React Native command line interface (CLI) for development, we must have the React Native development environment installed. For installation instructions, please follow [the official documentation](https://reactnative.dev/docs/environment-setup). You'll also want to make sure you select `react-native-cli` rather than `expo`. These instructions also walk you through starting your application, so if you are new to React Native, make sure you give them a read.
+Since we are going to use the React Native command line interface (CLI) for development, we must have the React Native development environment installed. For installation instructions, please follow [the official documentation](https://reactnative.dev/docs/environment-setup?guide=native). You'll also want to make sure you select `React Native CLI Quickstart` rather than `Expo Go Quickstart`. These instructions also walk you through starting your application, so if you are new to React Native, make sure you give them a read.
 
-We also need to install development environments for iOS, Android, or both. We are going to use [brew](https://brew.sh/) to install needed packages as well. If you are following along, make sure that `brew` is installed, or install the packages in a different way. 
+## React Native project setup
 
-### iOS environment
-
-First, we'll install watchman, which is used to automatically rebuild files when they change:
+To create a new React Native project, run this command:
 
 ```shell
-brew install watchman
-```
-
-Then we need to install the Xcode CLI tools, which are not normally present and can't be done with `brew`. To install, open Xcode and navigate to "Preferences" and then "Locations". Pick the Xcode version for command-line tools as shown in the screenshot below:
-
-{% include _image.liquid src="/assets/img/blogs/react-native-oauth/activate-xcode.png" alt="Activating Xcode." class="img-fluid" figure=false %}
-
-iOS is ready to go.
-
-### Android environment
-
-For Android, JDK 8 is required, as other versions may result in errors. We can download this version from the Oracle website or using `brew` as shown in the snippet below:
```

Comment:
If you remove this, how are they setting up the Android environment? Fine with getting rid of it, but if we do so, should have Android dev environment as a pre-req. 

Unless I'm missing something...

Reply:
Yes, with further thought, please add Android Studio as a pre-requisite and link to the install instructions elsewhere.

Reply:
I removed this portion of the blog post, because it's included in the [Setting up the development environment](https://reactnative.dev/docs/environment-setup?guide=native) guide for React Native